rotkehlchen/inquirer.py

```diff
@@ -517,25 +518,20 @@ def find_uniswap_v2_lp_price(
             block_identifier='latest',
         )
 
-    def find_curve_pool_price(
-        self,
-        lp_token: EthereumToken,
-    ) -> Optional[Price]:
+    def find_curve_pool_tokens_amount(
+            self,
+            lp_token_address: ChecksumEthAddress,
+    ) -> Optional[Dict[EthereumToken, Tuple[FVal, FVal]]]:
```

Comment:
You have renamed this but haven't adjusted the behavior in this file and in the tests

Reply:
No, `find_curve_pool_tokens_amount` is a new method. `find_curve_pool_price` still does the same.